Glideslope.json

```diff
@@ -37903,11 +37903,11 @@
    "identifier": "IBI",
    "associatedRunwayNumber": "18L",
    "frequency": 111300,
-   "elevation": 421,
-   "latitude": 50.35770035,
-   "longitude": 30.90455055,
-   "bearing": 177,
-   "glideslope": 3.00177,
+   "elevation": 427,
+   "latitude": 50.316386,
+   "longitude": 30.903153,
+   "bearing": 182.69,
+   "glideslope": 3.00,
```

Comment:
Hi! The glideslope here should be 3.0018269 here. The last 5 digits indicate the bearing :D
![image](https://user-images.githubusercontent.com/37546244/87382939-c9ec6580-c5ca-11ea-872c-5ca447c1a61d.png)